service/project.go

```diff
@@ -1009,3 +1015,87 @@ func verifyAliasExists(alias, projectId string, newAliasDefinitions []model.Proj
 	}
 	return false, nil
 }
+
+func bbProjectIsValid(projName string, proj evergreen.BuildBaronSettings) error {
+	webHook, _, _ := model.IsWebhookConfigured(projName, "")
+	webhookConfigured := webHook.Endpoint != ""
+
+	if !webhookConfigured && proj.TicketCreateProject == "" {
```

Comment:
Should this be an error still? I think this made sense when we had a list of build baron projects, but now this configuration just implies a project that doesn't intend to use the build baron. You can test this a bit on Evergreen local.

Reply:
Agreed, this should just have the error immediately return as nil.  Changed to this.

Reply:
Tested locally to confirm the modifyProject route works

Reply:
This is only not an error if webhook not configured, ticket create project is empty, AND ticket search project is empty. Right now this will allow users to just set ticket search project but not ticket create project, which isn't a valid configuration.